examples/platforms/efr32mg1/flash.c

```diff
@@ -0,0 +1,406 @@
+/*
```

Comment:
There are recent changes to this file in efr32m12 including a PR that is pending merge at the time I'm adding this comment. Please resync this file.